src/AppBundle/Entity/Client.php

```diff
@@ -716,4 +716,18 @@ public function getCoDeputies()
         }
         return array_values($coDeps);
     }
-}
+
+    /**
+     * @return array $submittedReports an array of submitted reports
+     */
+    public function getSubmittedReports()
```

Comment:
we get a list of submitted reports from lay_home page as well.
ReportController:84-91

What about some refactor to use this new method there?

a method getActiveReport  can be added alongside. And the the arsort can be left out in the controller

Reply:
Good call - Done!